NormalizedArrays/Sniffs/Arrays/CommaAfterLastSniff.php

```diff
@@ -147,6 +147,12 @@ public function process(File $phpcsFile, $stackPtr)
 
         $isComma = ($tokens[$lastNonEmpty]['code'] === \T_COMMA);
 
+        // If the closer in the same line than the last element, change the error code for multi-line arrays.
+        $sameLineCloser = ($tokens[$lastNonEmpty]['line'] === $tokens[$closer]['line']);
+        if ($errorCode === 'MultiLine' && $sameLineCloser) {
+            $errorCode .= 'CloserSameLine';
+        }
```

Comment:
```suggestion
        if ($errorCode === 'MultiLine'
            && $tokens[$lastNonEmpty]['line'] === $tokens[$closer]['line']
        ) {
            $errorCode .= 'CloserSameLine';
        }
```

(no need to declare a variable if the value is only used once)

I'd also suggest moving this block up to before the `$isComma` (and closer to the other error code logic).